pytorch_lightning/metrics/classification.py

```diff
@@ -507,29 +543,15 @@ def __init__(
 
             reduce_group: the process group to reduce metric results from DDP
         """
-        super().__init__(
+        super(TensorMetric, self).__init__(
```

Comment:
can't we just call fbeta init here with beta=1 and then overwrite the name? should be less code, no?

Reply:
I have changed it to your suggestion 👍